controller_report_xls/controllers/main.py

```diff
@@ -38,6 +38,96 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_integer(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_formatted_number(value):
+    set_sign = set([',', '.', '-'])
```

Comment:
is it really necessary the set if the values are wired? this values can come from the lang in Odoo.

![](http://screenshots.vauxoo.com/oem/4ec202-1346x868.png)